samples/net/wpanusb/src/wpanusb.c

```diff
@@ -417,6 +418,11 @@ void main(void)
 
 	radio_api = (struct ieee802154_radio_api *)ieee802154_dev->driver_api;
 
+	ret = sb_enable(NULL);
```

Comment:
What is sb_enable()? This may be an illustration why changes like this don't scale.

Reply:
Sorry for that. I have to remove it by mistake when rebasing.